data/netinfo.yml

```diff
@@ -0,0 +1,58 @@
+title: Sites breaking due to the lack of Network Information API support
+severity: low
+user_base_impact: small
+
+tags:
+  - mozilla-position-negative
```

Comment:
I wonder if we should lift this to a new top-level category (not for this PR ofc).

Reply:
I agree, separating those entries from the "regular" ones makes sense. These are things that we don't really plan on implementing, so treating those in a different way than our other breakage entries would be a good thing!